uniflow/flow/extract/extract_pdf_flow.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         super().__init__()
         self._process_pdf_op = ProcessPDFOp(
             name="process_pdf_op",
-            model=PreprocessModel(
+            model=LLMDataPreprocessor(
```

Comment:
nit: why not just call it `LLMPreprocessor` or `PreprocessorModel`.

Reply:
Please don't use the word Model... it's really confusing